pkg/storage/common.go

```diff
@@ -56,6 +57,19 @@ func GetManifestDescByReference(index ispec.Index, reference string) (ispec.Desc
 	return manifestDesc, false
 }
 
+func GetRoutePrefix(name string) string {
```

Comment:
there is also a `getRoutePrefix` function in `pkg/storage/storage_controller.go`, can you modify that one instead of adding this new function?